cmd/argocd/commands/app.go

```diff
@@ -1926,17 +1926,17 @@ func printAppResources(w io.Writer, app *argoappv1.Application, discoveryClient
 func mergeDuplicateClusterLevelResources(resourceStates []*versionedResourceState, discoveryClient discovery.DiscoveryInterface) []*versionedResourceState {
 	resourcesByKey := make(map[kube.ResourceKey]*versionedResourceState)
 	for _, resource := range resourceStates {
-		groupVersion := ""
+		APIVersion := ""
 		if resource.Group != "" {
-			groupVersion += (resource.Group + "/")
+			APIVersion += (resource.Group + "/")
 		}
-		groupVersion += resource.Version
+		APIVersion += resource.Version
 		// Here we are handling only the non-error case. The error case is ignored, because
 		// if there is an error, we assume it is not caused by technical reachability or request format
 		// but rather due to the API server not knowing the group/version which we are querying for.
 		// In that case, there is nothing sensible we can do, and we simply proceed by assuming that
 		// erroneous resource is correctly namespaced.
-		if apiResourceList, err := discoveryClient.ServerResourcesForGroupVersion(groupVersion); err == nil {
+		if apiResourceList, err := discoveryClient.ServerResourcesForGroupVersion(APIVersion); err == nil {
```

Comment:
Also rename the function.

Reply:
Sorry, cant do that, since that is the name of method in Kubernetes client-go library.

Reply:
Ah, didn't realize that. It's okay then.